.pre-commit-config.yaml

```diff
@@ -354,7 +354,8 @@ repos:
           ^.*pnpm-lock\.yaml$|
           .*/dist/.*|
           ^airflow-core/src/airflow/ui/src/i18n/locales/de/|
-          ^airflow-core/src/airflow/ui/src/i18n/locales/pl/
+          ^airflow-core/src/airflow/ui/src/i18n/locales/pl/ |
+          ^airflow-core/src/airflow/ui/src/i18n/locales/tr/
```

Comment:
Maybe we could use something like (didn't test it):
`^airflow-core/src/airflow/ui/src/i18n/locales/(?!en/)$`

So it would exclude all languages but English?

Reply:
Let me check this, it would be a lot easier

Reply:
Thanks, Shahar, for handling this case!